src/player.cpp

```diff
@@ -3392,8 +3393,8 @@ int player::print_aim_bars( WINDOW *w, int line_number, item *weapon, Creature *
     // This simplifies the calculation greatly, that's intentional.
     const std::array<std::pair<double, char>, 3> ratings =
         {{ std::make_pair(0.1, '*'), std::make_pair(0.4, '+'), std::make_pair(0.6, '|') }};
-    const std::string confidence_label = _("Confidence: ");
```

Comment:
Why this spacing change?  The colon is to set of the label vs the bar, with a space to make it extra clear, putting the colon immediately next to the bar makes it look like the colon is part of the bar.